app/ui/design-system/src/lib/Components/NavigationBar/DesktopMenuItem.tsx

```diff
@@ -61,15 +61,15 @@ export function DesktopMenuItem({ divider, ...props }: DesktopMenuItemProps) {
             leaveFrom="transform opacity-100 scale-100"
             leaveTo="transform opacity-0 scale-95"
           >
-            <Popover.Panel className="fixed top-[96px] left-0 right-0 bottom-0 z-40 origin-top-right">
+            <Popover.Panel className="fixed top-[90px] left-0 right-0 bottom-0 z-40 origin-top-right">
```

Comment:
Can these use the `NAV_HEIGHT` constant so they stay in sync? It may need to be updated to 90px

Reply:
Ah didn't even realize that existed. Will update that.